server/src/main/java/com/linecorp/centraldogma/server/storage/repository/Repository.java

```diff
@@ -568,9 +568,4 @@ default <T> CompletableFuture<MergedEntry<T>> mergeFiles(Revision revision, Merg
      * {@link RepositoryListener#pathPattern()} are pushed to this {@link Repository}.
      */
     void addListener(RepositoryListener listener);
-
-    /**
-     * Removes the {@link RepositoryListener} from this {@link Repository}.
-     */
-    void removeListener(RepositoryListener listener);
```

Comment:
This API was removed to simplify the implementations. Since this API is designed for internal usage, I don’t think the removal will cause any breaking changes.